src/tests/inventorySorting.spec.js

```diff
@@ -0,0 +1,69 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+
+const cleanPrices = (prices) => prices.map(price => parseFloat(String(price).replace(/$/g, '')));
+
+const sortPrices = (prices, order) => {
+    const cleanedPrices = cleanPrices(prices);
+    return order === 'asc' ? cleanedPrices.sort((a, b) => a - b) : cleanedPrices.sort((a, b) => b - a);
+};
+
+test.beforeEach(async (
+    /** @type {{ app: import('../pages/Application').Application }} */{ app },
+) => {
+    await app.login.navigate();
+    await app.login.performLogin('standard_user', 'secret_sauce');
```

Comment:
Please, create a separate file with credentials and use them inside all tests you need. 
To use it like: 
```
await app.login.performLogin(user.username, user.password);
```